tests/spec/unit/templates/cordova/lib/prepare.spec.js

```diff
@@ -1414,7 +1414,7 @@ describe('Testing prepare.js:', () => {
 
     describe('createResourceMap method', () => {
         let createResourceMap;
-        let shellLsSpy;
+        let existsSyncSpy;
```

Comment:
Is this still used outside of `beforeEach`?

Reply:
removed